conceptpower/classes.py

```diff
@@ -157,4 +159,65 @@ def get_type(self, uri):
                 data['supertype_id'] = snode.get('supertype_id')
                 data['supertype_uri'] = snode.get('supertype_uri')
         return data
-        
\ No newline at end of file
+
+    def create(self, user, password, word, pos, conceptlist, description, type, synonym_ids, equal_uris, similar_uris):
+        """
+        Adds a concept to the conceptlist by using rest api
+
+        Parameters
```

Comment:
In these descriptions you use the term "word" a lot  (e.g. 'name of the word') -- we're adding a concept not a word. Let's keep the terminology consistent. Also, descriptions should be grammatically correct (start with a capital letter, end with a period).